### PR TITLE
Make JToolbar more semantic.

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: libraries/legacy/toolbar/button/help.php [hunk 1]
```diff
@@ -42,11 +42,11 @@ public function fetchButton($type = 'Help', $ref = '', $com = false, $override =
 		$class = $this->fetchIconClass('help');
 		$doTask = $this->_getCommand($ref, $com, $override, $component);
 
-		$html = "<a href=\"javascript:void(0)\" onclick=\"$doTask\" rel=\"help\" class=\"toolbar\">\n";
```
Comment (ianmacl) [inline libraries/legacy/toolbar/button/help.php:45]: What is this supposed to do?  It doesn't look like anything happens if you click on this button.  Did I miss something?

**Comment by ianmacl** [comment]
https://tinker.io/f7b7e

Demo link for using button instead of href.  If people can try it in a couple of browsers it would be helpful.  You want to make sure that the alert appears and that the message is logged to the console.

**Comment by elinw** [comment]
Works for me in Chrome on Ubuntu.

File: libraries/legacy/toolbar/button/help.php (hunk 1 above)
Comment (realityking) [inline libraries/legacy/toolbar/button/help.php:45]: You're right, I accidentally removed the onclick attribute. I'll add it back.

Comment (realityking) [inline libraries/legacy/toolbar/button/help.php:45]: Fixed.

**Comment by realityking** [comment]
Closing since JToolbar has been moved to the CMS.

**Pull request closed without merging**